### DISCUSSION TITLE
Colors for different kinds of character

### DISCUSSION THREAD

**Discussion by snhv** [lockerpm/.github#2] [Ideas]
As titled, we should have different colors for different kinds of characters in the password field: numbers, capitalized text, normal text, special characters, in order to avoid confused characters, such as capitalized i and normal l or number 0 and capitalized o.
Thanks.

**Comment by issimonsi** [comment]
Hi @snhv ,

Ensuring visual distinction among different types of characters in the password field can certainly enhance usability and minimize confusion. I'll make a note to add this feature to our backlog for implementation. We'll prioritize it based on our capacity and existing roadmap. Thanks for the suggestion!
